egs2/TEMPLATE/tts1/tts.sh

```diff
@@ -426,8 +439,49 @@ if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
     _train_dir="${data_feats}/${train_set}"
     _dev_dir="${data_feats}/${dev_set}"
     log "Stage 5: TTS Training: train_set=${_train_dir}, dev_set=${_dev_dir}"
-
+    
     _opts=
+    if [ -n "${teacher_model_path}" ] && echo "${train_config}" | grep -q "fastspeech"; then
+      # Setup feature and duration for fastspeech knowledge distillation training
+      teacher_expdir=$(dirname "${teacher_model_path}")
+      teacher_outdir=outputs_$(basename ${teacher_model_path})_$(basename ${teacher_decode_config%.*})
+      teacher_outdir=${teacher_expdir}/${teacher_outdir}
+      if [ ! -e ${teacher_outdir}/.done ]; then
+        local/setup_knowledge_dist.sh \
+          --nj ${nj} \
```

Comment:
Don't use local script in tts.sh.
If necessary, create it in script/

Reply:
I moved local/setup_knowledge_dist.sh to TEMPLATE/tts1/scripts/utils/ .